pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -42,6 +42,25 @@
 from ..tools import ToolDefinition
 from . import Model, ModelRequestParameters, StreamedResponse, check_allow_model_requests, download_item, get_user_agent
 
+
+def _map_anthropic_finish(raw: str | None) -> str | None:
```

Comment:
This should return `OtelFinishReason`.

Reply:
Let's include `finish_reason` in the name here

Reply:
ok